.vscode/tasks.json

```diff
@@ -0,0 +1,23 @@
+{
+    "tasks": [
+        {
+            "type": "cppbuild",
+            "label": "build cloe",
+            "command": "make",
+            "args": [
+                "all",
+                "CONAN_OPTIONS=\"-c tools.build:skip_test=1\"",
```

Comment:
Maybe make two different tasks, one with and one without unit tests.

During many development, you want to ensure unit tests are green before trying any other things. They are the *first line of defence*, so to speak.